xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd

```diff
@@ -111,6 +111,47 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Maximum number of passengers who can traverse PATH LINK per minute.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="Width" type="LengthType" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Width of pathLink.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="FlooringType" type="FlooringTypeEnumeration" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Type of flooring of the walking surface.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			
+			<xsd:element name="RightSideBorder" type="BorderTypeEnumeration" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Type of border on the right side ((in the direction of the pathLink way).</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="LeftSideBorder" type="BorderTypeEnumeration" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Type of border on the left side ((in the direction of the pathLink way).</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>			
+			<xsd:element name="TiltAngle" type="xsd:integer" minOccurs="0">
```

Comment:
@nick-knowles I notice that neighter TiltAngle, CodedTilt are part of the current master, while TiltAngle seem to appear in PathNavigationGroup. Please advise.

Reply:
Shouldn't this be called `Slope`?

Reply:
@nick-knowles @Aurige what does transmodel state?

Reply:
Transmodel doesn't (yet?) go to that level of details on all attributes related to accessibility. But we definitely need the Tilt information on NavigationPath

Reply:
I hope this shameless plug is appropriate: If you or somebody you know needs consulting with best practices regarding accessibility criteria, let me know, we have been working on this topic for more than a decade.

We're Sozialhelden e.V., a non-profit, building [Wheelmap.org](https://wheelmap.org), [A11yJSON](https://a11yjson.org),  and founded the [W3C LDA CG](https://www.w3.org/groups/cg/lda).

As the number of accessibility criteria like this is going to grow in data formats, there might be hundreds of properties like this soon, which means some kind of alignment will help everyone involved. Just sign up for the W3C LDA CG, and if you need specific consulting for your specs, send us a mail to developers@sozialhelden.de, we're happy to get involved and there might be some things that you find useful :)

(CC @holgerd)

Reply:
@opyh are you already participating in the actual discussion on basecamp?

Reply:
I'll send you a PM. Thanks!